baseframe/forms/fields.py

```diff
@@ -305,8 +305,15 @@ def __init__(self, *args, **kwargs):
         self.usermodel = kwargs.pop('usermodel')
         self.lastuser = kwargs.pop('lastuser')
         self.separator = kwargs.pop('separator', ',')
+        if self.lastuser:
+            self.autocomplete_endpoint = self.lastuser.endpoint_url(self.lastuser.getuser_autocomplete_endpoint)
+            self.getuser_endpoint = self.lastuser.endpoint_url(self.lastuser.getuser_userids_endpoint)
+        else:
+            self.autocomplete_endpoint = kwargs.pop('autocomplete_endpoint')()
+            self.getuser_endpoint = kwargs.pop('getuser_endpoint')()
         super(UserSelectFieldBase, self).__init__(*args, **kwargs)
 
+
```

Comment:
Should be one blank line between methods inside a class. Two blank lines at the module level.